docs/resources/server/enchantments/builtin.md

```diff
@@ -0,0 +1,183 @@
+# Built-In Enchantment Effect Components
+
+Vanilla Minecraft provides numerous different types of Enchantment Effect Components for use in [enchantment] definitions. This article will explain each, including their usage and in-code definition.
+
+## Value Effect Components
+[Value Effect Components] are used for enchantments that alter a numerical value somewhere in the game, and are implemented by the class `EnchantmentValueEffect`. 
```

Comment:
Consider replacing the inline links with something like this:
```
_See also: [Value Effect Components] on the Minecraft Wiki_
```
Similar for the other links below.